config.json

```diff
@@ -120,10 +120,33 @@
           "characters",
           "strings"
         ]
+      },
+      {
+        "slug": "anagram",
+        "name": "Anagram",
+        "uuid": "44141229-e4ba-475e-9340-bd7663884da5",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1,
+        "topics": [
+          "characters",
+          "strings"
+        ]
       }
     ]
   },
   "concepts": [],
   "key_features": [],
-  "tags": []
+  "tags": [
+    "paradigm/functional",
+    "paradigm/array",
+    "execution_mode/interpreted",
+    "platform/windows",
+    "platform/mac",
+    "platform/linux",
+    "runtime/interpreter_or_lib",
+    "used_for/data_analysis",
+    "used_for/mathematics",
+    "used_for/finance"
```

Comment:
I admit I just made up these tags; is there an official list that should be used?

Reply:
Ran `configlet` and looked up what currently valid tags to use instead; updated.